src/components/middle/composer/DropArea.tsx

```diff
@@ -71,7 +71,10 @@ const DropArea: FC<OwnProps> = ({
     const { target: fromTarget, relatedTarget: toTarget } = e;
 
     // Esc button pressed during drag event
-    if ((fromTarget as HTMLDivElement).matches('.DropTarget, .DropArea') && !toTarget) {
+    if (
+      (fromTarget as HTMLDivElement).matches('.DropTarget, .DropArea') && !toTarget ||
```

Comment:
Our ESLint rules do not allow mixing `&&` and `||`.

Reply:
hello thanks for your comment,
i edited the if condition in last commit.

is it still consider mixing || and  && in the edited condition?

Reply:
Thanks for your PR. You can check ESLint with `npm run check`.

Reply:
i committed to this branch according to your suggestion Alexander, your suggestion was a better approach.
thanks for your feedbacks